lxd/container.go

```diff
@@ -519,7 +519,7 @@ func instanceCreateFromImage(d *Daemon, args db.InstanceArgs, hash string, op *o
 		if op != nil {
 			tracker = &ioprogress.ProgressTracker{
 				Handler: func(percent, speed int64) {
-					// tomp TODO should the container reference here be removed?
+					// tomp TODO should the container reference here be changed to instance?
```

Comment:
@stgraber should the progress meta data label be changed to refer to instance rather than container?

Reply:
We don't seem to be looking at those names directly anywhere in the client, so yeah, I think it's safe to update them.